packages/mainmenu-extension/src/index.ts

```diff
@@ -485,6 +485,18 @@ function createTabsMenu(app: JupyterLab, menu: TabsMenu): void {
     return { command: commandID };
   };
 
+  let previousId = '';
+
+  // Command to toggle between the current
+  // tab and the last modified tab.
+  const commandID = 'application:activate-previous-active-tab';
+  if (!commands.hasCommand(commandID)) {
+    commands.addCommand(commandID, {
+      label: 'Activate Previous Active Tab',
```

Comment:
I think this label is kind of confusing. Perhaps something like "Activate Previously Used Tab"?